ci_framework/roles/edpm_build_images/tasks/run.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Build EDPM hardened uefi image
+  when:
+    - (cifmw_edpm_build_images_hardened_uefi | bool) or (cifmw_edpm_build_images_all | bool)
+  environment:
+    DIB_LOCAL_IMAGE: "{{ cifmw_edpm_build_images_dib_local_image_path }}/{{ cifmw_edpm_build_images_base_image_name }}"
+    DIB_YUM_REPO_CONF: "{{ cifmw_edpm_build_images_dib_yum_repo_conf | join(' ') }}"
+    ELEMENTS_PATH: "{{ cifmw_edpm_build_images_elements | join(':') }}"
+  ansible.builtin.shell: |
+    diskimage-builder images/edpm-hardened-uefi-{{ distro_name }}-{{ distro_version }}-stream.yaml > {{ cifmw_edpm_build_images_artifacts }}/edpm_images/edpm_hardened_uefi_image_build.log
```

Comment:
1) imho.. logs should go in logs directory and not to the artifacts directory
2) Please also redirect stderr to a file.

you can refer tripleo image build example:-

https://logserver.rdoproject.org/openstack-periodic-integration-stable1/opendev.org/openstack/tripleo-ci/master/periodic-tripleo-centos-9-buildimage-overcloud-hardened-uefi-full-wallaby/0e8ddf7/build_images.sh

~~~
openstack --debug overcloud image build --image-name overcloud-hardened-uefi-full \
    $CONFIG_FILES > /home/zuul/workspace/build.log 2> /home/zuul/workspace/build-err.log
~~~

Reply:
+1 I see logs more related to `~/ci-framework-data/logs`, is the place we use for deployment logs too.